### ISSUE TITLE
Thickened Descemet membrane

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7582]
**Preferred term label:**

**Synonyms**
Thick Descemet membrane

**Definition (free text, please give PubMed ID)**
An elevation in the thickness of Descemet's membrane.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:271310 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@Sukram is this a good term?

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000312
- label: Thickened Descemet membrane
- parents: [Abnormal Descemet membrane morphology (HP:0011490)](https://hpo.jax.org/app/browse/term/HP:0011490)
- pmid: 
- def: The thickness of the Descement membrane is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7582
- synonyms: Thick Descemet membrane(NONE)
